docs/API.md

```diff
@@ -1395,12 +1422,31 @@ Example: `GET trusty/wordpress-42/meta/revision-info`
 }
 ```
 
+Example: `GET wordpress/meta/revision-info`
+
+```json
+{
+    "Revisions": [
+        "cs:trusty/wordpress-43",
+        "cs:trusty/wordpress-42",
+        "cs:trusty/wordpress-41",
+        "cs:trusty/wordpress-39"
+        "cs:precise/wordpress-41",
+        "cs:precise/wordpress-39"
+    ]
+}
+```
+
 #### GET *id*/meta/id
 
 The `id` path returns information on the charm or bundle id, split apart into
 its various components, including the id itself. The information is exactly
 that contained within the entity id.
 
+The request URL may omit series. For charms which support multiple series
```

Comment:
Quick response to this one point - old clients do need to be able to deploy multi series charms. There was discussion that when a multi-series charm is published, separate entity entries would be created for each supported series. We you say code will break, do you mean charmstore code? Could it not be changed to be able to look for an entity with series in the id and revert to an id without series if not found?